### ISSUE TITLE
Finalize Caret Preprocessing : Normalization Methods

### ISSUE THREAD

**Issue by antoinecarme** [antoinecarme/caret2sql#22]
Add support for normalization methods, Following the issue :

Finalize Caret Preprocessing #18 

support for "spatialSign", "range" methods

These two methods have an equivalent in scikit-learn (have a SQL generator) : 

spatialSign <=> sklearn.preprocessing.Normalizer
range <=> sklearn.preprocessing.MinMaxScaler

https://github.com/topepo/caret/blob/master/pkg/caret/R/preProcess.R

**Comment by antoinecarme** [comment]
Added two notebooks for range and spatialSign
